.github/workflows/rebuild-latest-stable-image.yml

```diff
@@ -0,0 +1,57 @@
+name: Rebuild latest release image version
+
+on:
+  schedule:
+    - cron: 0 0 * * 1
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
```

Comment:
Is a push on `main` equivalent to the merge of a PR?

Reply:
I'm not sure how this repository is set up, in general yes, but if it allows pushing commits to main, the pipeline would start as well.